src/renderer/containers/Table/Editors/DateEditor/index.tsx

```diff
@@ -41,7 +41,7 @@ export default function DateEditor({
         (e.relatedTarget.tagName !== "LI" ||
           e.relatedTarget.attributes.getNamedItem("role")?.value !== "button"))
     ) {
-      handleCommit();
+      handleCommit(value);
```

Comment:
Was the fix just to pass `value` to the callback, or was it also changing the `Date` type to `Moment`?

Reply:
The fix was to pass the value into the callback.

Changing the type to `Moment` just helped make that cleaner, because it enabled me to pass `value` as a prop to `handleCommit` without having to make `handleCommit` handle both `Date` and `Moment` parameters. And the type of `value` was already being changed back and forth from date to moment in a couple of spots anyways.